sparkly/reader.py

```diff
@@ -213,71 +217,110 @@ def kafka(self,
               schema=None,
               port=9092,
               parallelism=None,
-              options=None):
+              options=None,
+              include_meta_cols=None):
```

Comment:
also renamed this from `verbose` to `include_meta_cols`; lmk what you think

Reply:
Looks great, thanks